frontend/src/components/worksheets/BundleDetail/BundleDetail.jsx

```diff
@@ -204,6 +204,25 @@ const BundleDetail = ({
         }
     };
 
+    /**
+     * This helper syncs the state info that is used to render the bundle row
+     * with the state info that is passed down into the bundle detail sidebar.
+     *
+     * This enables the bundle row and the bundle detail sidebar to show the
+     * exact same state information.
+     */
+    const syncBundleStateInfo = () => {
```

Comment:
This only is called once, so why not just inline?

Reply:
I feel like wrapping the logic in a named function makes it easier to figure out for someone else who might stumble across the code later.